### PR TITLE
translations refinements

### PR DESCRIPTION
**Pull request by dzhaniivanov**
#1724  + translation bugs caught on the fly from the team.

task

### CODE REVIEW

File: public/locales/bg/one-time-donation.json [hunk 1]
```diff
@@ -49,9 +49,9 @@
   "third-step": {
     "title": "Как желаете да дарите?",
     "card": "Карта",
-    "card-include-fees": "Искам да покрия таксата за карта издадена в: ",
-    "card-fees": "При дарение с карта използваме услугите на Stripe, като за всеки трансфер Stripe начисляват такса според региона на картоиздателя ви. За да се ориентирате за нетното дарение, след като изберете желаната сума, ще ви покажем изчислената такса. Повече за таксите от Страйп вижте тук: ",
-    "card-calculated-fees": "За вашия превод от <strong>{{totalAmount}}</strong>, таксата на Stripe ще е <strong>{{fees}}</strong>, а кампанията ще получи <strong>{{amount}}</strong>",
+    "card-include-fees": "Съгласен съм да покрия таксата за превода, така че да се получи чистата вума, която съм избрал/а като дарение, за карта издадена в: ",
```
Comment (ani-kalpachka) [inline public/locales/bg/one-time-donation.json:52]: Did you mean 'чистата сума'?

File: public/locales/bg/one-time-donation.json [hunk 2]
```diff
@@ -70,7 +70,7 @@
     "reason-donation": "Oснование за превод:",
     "message-warning": "Ако не въведете точно основанието, може да не успеем да разпределим парите към предназначената кампания.",
     "recurring-donation-title": "Месечно дарение",
-    "recurring-donation-info": "Дарявай повторно всеки месец тази сума до края на кампанията! Може да се откажете по всяко време във профила си."
+    "recurring-donation-info": "Желая да дарявам същата сума всеки месец до края на кампанията.Може да се откажете по всяко време в профила си: "
```
Comment (ani-kalpachka) [inline public/locales/bg/one-time-donation.json:73]: Can we keep the space between these two setences please?

File: public/locales/bg/one-time-donation.json (hunk 1 above)
Comment (dzhaniivanov) [reply public/locales/bg/one-time-donation.json:52]: Yup, copy/paste ☺️ sorry

**Review by ani-kalpachka — CHANGES_REQUESTED** [review]
Good improvements!
Can you please check the minor issues related to the translations?

**Review by ani-kalpachka — APPROVED** [review]

**Pull request merged**